src/Stratis.Bitcoin.Features.Wallet/WalletSyncManager.cs

```diff
@@ -248,5 +268,15 @@ public virtual void SyncFromHeight(int height)
 
             this.logger.LogTrace("(-)");
         }
+
+        /// <inheritdoc />
+        public void Dispose()
+        {
+            this.logger.LogTrace("()");
+
+            this.blocksQueue?.Dispose();
```

Comment:
I don't think blocksQueue can ever be null.

Reply:
ok